package.json

```diff
@@ -18,7 +18,7 @@
     "lint": "eslint .",
     "flow": "flow check",
     "danger": "danger",
-    "fix": "eslint . --fix",
+    "fix": "yarn eslint . --fix",
```

Comment:
Ok cool, otherwise everything else looks good so let's merge after removing that